### ISSUE TITLE
Opentracing active_span not working without starting new child span

### ISSUE THREAD

**Issue by AdrianLC** [DataDog/dd-trace-py#1153]
### Which version of dd-trace-py are you using?
ddtrace[opentracing]==0.31.0

### Which version of the libraries are you using?
python 3.8
Django==2.2.6
channels==2.3.1
uvicorn==0.10.3
uvloop==0.14.0rc2
datadog==0.31.0

### How can we reproduce your problem?
When calling `opentracing.global_tracer().active_span` with automatic instrumentation of Django requests enabled. 

These are our Datadog related settings:
```python

    INSTALLED_APPS.append('ddtrace.contrib.django')

    from ddtrace import patch
    from ddtrace.filters import FilterRequestsOnUrl
    from ddtrace.opentracer import Tracer, set_global_tracer

    DATADOG_TRACE = {
        'DEFAULT_SERVICE': DATADOG_SERVICE_NAME,
        'AUTO_INSTRUMENT': True,
        'INSTRUMENT_DATABASE': False,
        'INSTRUMENT_CACHE': False,
        'INSTRUMENT_TEMPLATE': False,
        'AGENT_HOSTNAME': env('DD_AGENT_HOST'),
        'AGENT_PORT': env.int('DD_AGENT_PORT', default=8126),
    }

    set_global_tracer(
        Tracer(
            DATADOG_SERVICE_NAME,
            config={
                'settings': {
                    'FILTERS': [
                        FilterRequestsOnUrl(r'^http://.+/_health$'),
                    ],
                }
            }
        )
    )

    _DD_PATCHES = (
        'psycopg',
        'celery',
        'redis',
        'requests'
    )

    patch(**{module: True for module in _DD_PATCHES})
```
We have not done any special configs on Datadogs' context providers or Opentracing's scope managers. The debugger shows that these are `ContextVarContextManager` for Datadog and `ThreadLocalScopeManager` for Opentracing. All our code runs on ThreadpoolExecutors created automatically by django-channels and we have not written any asyncio code yet.

### What is the result that you get?
`active_span` returns None even though there is a DataDog span that is active for the Django request.

### What is result that you expected?
I would expect that that the current Datadog Span is available as an opentracing span without the need to create an additional child span.

**Comment by AdrianLC** [comment]
Possibly related?? https://github.com/DataDog/dd-trace-py/pull/838

**Comment by Kyle-Verhoog** [comment]
Hi @AdrianLC, thanks for reporting.

It looks like our opentracing tracer did not support getting the active span properly. It would work if using opentracing code but would not for the datadog instrumentation.

I've added a fix for this in #1395.
